### ISSUE TITLE
list of file types

### ISSUE THREAD

**Issue by Perzan** [MicrosoftDocs/windowsserverdocs#3905]
Is there a list of file types to use in this command?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6ddbfcb0-76bc-5263-5754-29f5d5539522
* Version Independent ID: ac32f8da-db57-0c70-b9a8-a1f0030495e8
* Content: [assoc](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/assoc#feedback)
* Content Source: [WindowsServerDocs/administration/windows-commands/assoc.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/administration/windows-commands/assoc.md)
* Product: **windows-server**
* Technology: **manage-windows-commands**
* GitHub Login: @coreyp-at-msft
* Microsoft Alias: **coreyp**

**Comment by illfated** [comment]
Fair question, but I think you are missing the point of this command line (CMD) command. This command (**assoc**) is more or less the command line equivalent of "Choose default apps by file type":

Basically, you use the command to tell Windows which program to use with _any_ of your file name extensions. I think we are limited to 1 extension + 1 association per command, but I have also not checked if making a new association is equal to setting which program opens files with that file name extension.

See for yourself by running the command **`assoc`** in a CMD window. That should give you some more insight, I think.

**Comment by illfated** [comment]
TL;DR: Run the command **`assoc`** in a CMD window (make sure you have extended the CMD window to max screen height to enjoy the long, long list of file extensions and their file type definitions).

**Comment by dknappettmsft** [comment]
This has been answered above.

#please-close
